### PR TITLE
FAL-162: enable RDS autoscaling with max_allocated_storage

### PR DESCRIPTION
**Pull request by eLRuLL**
Enables RDS autoscaling

Reviewer:

- [ ] @pomegranited

### CODE REVIEW

File: modules/services/sql/variables.tf [hunk 1]
```diff
@@ -6,6 +6,9 @@ variable "engine_version" {
   default = "5.6.48"
 }
 variable "allocated_storage" {}
+variable "max_allocated_storage" {
+  default = 100
+}
```
Comment (pomegranited) [inline modules/services/sql/variables.tf:11]: Could you add a comment that indicates that this [enables autoscaling when `allocated_storage` < `max_allocated_storage`](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/db_instance#storage-autoscaling)?

That way, anyone who's using this template will know how to disable autoscaling if they want to.

Comment (pomegranited) [reply modules/services/sql/variables.tf:11]: On second thought, these docs should go in the [README](https://github.com/open-craft/terraform-scripts/tree/main/modules/services/sql#input)

Comment (eLRuLL) [reply modules/services/sql/variables.tf:11]: I updated the readme, I think it's actually that it needs to be 0 to be disabled :+1:

Comment (pomegranited) [reply modules/services/sql/variables.tf:11]: Ah cool yes, the terraform docs agree; 0 disables autoscaling. Thanks for updating the docs!

**Review by pomegranited — APPROVED** [review]
:+1:

- [x] I tested this:
   * Ran `terraform plan` with the latest linked client's secure repo, and ensured that no changes were proposed.
   * Checked the RDS instance and replica to ensure that autoscaling is enabled.
- [x] I read through the code
- [x] <strike>I checked for accessibility issues</strike> N/A
- [x] Includes documentation
- [x] I made sure any change in configuration variables is reflected in the corresponding client's `configuration-secure` repository.

**Pull request merged**